addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -175,7 +175,7 @@ def _l10n_in_edi_post_invoice(self, invoice):
                 return {invoice: {
                     "success": False,
                     "error": error_message,
-                    "blocking_level": ("404" in error_codes) and "warning" or "error",
+                    "blocking_level": "warning" if any(code in error_codes for code in ('404', 'timeout')) else "error",
```

Comment:
@jva-odoo Ok, but what in 18.1?

Reply:
Send And Print not work like that so we need add new field in Send and Print with name like `Error Level` or `Blocking level` and base on that we can retry from cron of Send and Print

Reply:
There is a PR here https://github.com/odoo/odoo/pull/224267/files working through sending_data